src/shared-components/FormView/BrokerProperties/ConfigurationPage/AcceptorsConfigPage/AcceptorConfigSection/AcceptorConfigPage/AcceptorConfigPage.tsx

```diff
@@ -176,6 +178,10 @@ export const AcceptorConfigPage: FC<AcceptorProps> = ({
   ];
 
   const { t } = useTranslation();
+  useEffect(() => {
+    setPort(initialPort);
+  }, [initialPort]);
+
```

Comment:
Please don't use `useEffect` for this, refer to https://react.dev/learn/you-might-not-need-an-effect for more details about why this isn't a good pattern.